src/extensions/renderer/base/coord-ele-math/edge-control-points.js

```diff
@@ -756,7 +735,7 @@ BRp.findEdgeControlPoints = function( edges ){
     let srcIndex = src.poolIndex();
     let tgtIndex = tgt.poolIndex();
 
-    let pairId = [ srcIndex, tgtIndex ].sort();
+    let pairId = `${srcIndex}_${tgtIndex}_${edgeIsUnbundled}`;
```

Comment:
This needs to have the IDs sorted, because a bezier bundle can contain edges that go in either direction, e.g.

E1: A -> B
and
E2: B ->A
could both be in the same bundle